configure

```diff
@@ -115,15 +126,30 @@ EOF
   fi
 }
 
+_disable_readline() {
+  if test "$FLAG_with_readline" = 1; then
+    die 'readline was not detected on the system (--with-readline passed).'
+  fi
+  echo 'HAVE_READLINE='
+}
+
 # Write a shell script to standard out with variables, or fail.
 detect_readline() {
-  if cc_quiet build/detect-readline.c -l readline; then
-    echo 'HAVE_READLINE=1'
-  else
-    if test "$FLAG_with_readline" = 1; then
-      die 'readline was not detected on the system (--with-readline passed).'
+  if is_macos; then
+    if quiet command -v brew && quiet brew --prefix readline; then
```

Comment:
Why is it necessary to call out to `brew`?  Is it because the C compiler is installed with Xcode but the libraries are installed with homebrew?

Why not just have people either:

1) Install a homebrew package of OSH, which already depends on readline.  No compilation necessary.
2) compile readline themselves and compile OSH against it.

It seems weird to install readline through `brew`, and then compile OSH manually.  It seems like both should be installed in the same fashion -- either with or without `brew`.